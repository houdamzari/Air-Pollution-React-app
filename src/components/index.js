export { default as SingleCity } from "./SingleCity/SingleCity";
export { default as Container } from "./Container/Container";
export { default as Grid } from "./Grid/Grid";
export { default as Search } from "./Search/Search";
export { default as Back } from "./Back/Back";
