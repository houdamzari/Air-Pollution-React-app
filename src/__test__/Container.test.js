import "@testing-library/jest-dom";
import { Provider } from "react-redux";
import { render } from "@testing-library/react";
import { Container } from "../components";
import renderer from "react-test-renderer";
import { BrowserRouter } from "react-router-dom";

describe("Test Container Component", () => {
  test("Container component should match with snapshot", () => {
    const container = renderer
      .create(
        <BrowserRouter>
          <Container />
        </BrowserRouter>
      )
      .toJSON();

    expect(container).toMatchSnapshot();
  });
});
