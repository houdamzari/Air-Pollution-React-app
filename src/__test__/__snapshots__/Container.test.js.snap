// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Container Component Container component should match with snapshot 1`] = `
<div
  className="container"
/>
`;
